### PR TITLE
feat: Detect IQ Server Capabilities

### PR DESCRIPTION
**Pull request by maurycupitt**
Detect IQ Server Capabilities to highlight available features and possible upgrade paths.

### CODE REVIEW

File: src/components/Options/IQServer/IQServerOptionsPage.tsx [hunk 1]
```diff
@@ -163,107 +175,278 @@ export default function IQServerOptionsPage(props: IqServerOptionsPageInterface)
...
+                    <NxGrid.Row>
+                        <section className='nx-grid-col nx-grid-col--'>
                             <p className='nx-p'>
-                                <strong>2)</strong> {_browser.i18n.getMessage('OPTIONS_PAGE_SONATYPE_POINT_2')}
+                                <strong>1)</strong> {_browser.i18n.getMessage('OPTIONS_PAGE_SONATYPE_POINT_1')}
                             </p>
+
                             <div className='nx-form-row'>
-                                <NxFormGroup label={_browser.i18n.getMessage('LABEL_USERNAME')} isRequired>
+                                <NxFormGroup label={`URL`} isRequired>
```
Comment (madpah) [inline src/components/Options/IQServer/IQServerOptionsPage.tsx:348]: Label is not translated @maurycupitt

File: src/components/Popup/IQServer/LicensingPage/LicensingPage.tsx [hunk 2]
```diff
@@ -51,16 +51,16 @@ function IqLicensePage() {
         const declaredLicenses = licenseData?.declaredLicenses
         const effectiveLicenses = licenseData?.effectiveLicenses
 
-        if (popupContext.iq?.componentLegalDetails === undefined) {
-            return <Puff />
-        }
+        // if (popupContext.iq?.componentLegalDetails === undefined) {
```
Comment (madpah) [inline src/components/Popup/IQServer/LicensingPage/LicensingPage.tsx:54]: Remove commented code @maurycupitt

File: src/components/Options/IQServer/IQServerOptionsPage.tsx [hunk 3]
```diff
@@ -163,107 +175,278 @@ export default function IQServerOptionsPage(props: IqServerOptionsPageInterface)
...
+                                </center>
+                            </div>
+                        </div>
+                        <div className='nx-grid-col nx-grid-col-33'>
+                            <div>
+                                <center>
+                                    <img
+                                        src='/images/add-on-sonatype-icon-water.png'
+                                        width='50'
+                                        alt=''
```
[the remark this answers is not in the document]
Comment (maurycupitt) [reply src/components/Options/IQServer/IQServerOptionsPage.tsx:285]: Yes

File: src/components/Options/IQServer/IQServerOptionsPage.tsx [hunk 4]
```diff
@@ -163,107 +175,278 @@ export default function IQServerOptionsPage(props: IqServerOptionsPageInterface)
...
+            <NxTile>
+                <NxTile.Content>
+                    <NxGrid.Row>
+                        <div className='nx-grid-col nx-grid-col-33'>
+                            <div>
+                                <center>
+                                    <img
+                                        src='/images/sonatype-lifecycle-icon.png'
+                                        width='50'
+                                        alt=''
```
[the remark this answers is not in the document]
Comment (maurycupitt) [reply src/components/Options/IQServer/IQServerOptionsPage.tsx:254]: Yes

File: src/components/Options/IQServer/IQServerOptionsPage.tsx (hunk 1 above)
Comment (maurycupitt) [reply src/components/Options/IQServer/IQServerOptionsPage.tsx:348]: It is in the messages.json - is that what you mean?

Comment (madpah) [reply src/components/Options/IQServer/IQServerOptionsPage.tsx:348]: It is not currently in messages - but should be?

File: public/_locales/en/messages.json [hunk 5]
```diff
@@ -331,12 +351,20 @@
         "message": "Request Support",
         "description": "Sidebar Footer Link in Options Page"
     },
+    "SUPPORTED": {
```
[the remark this answers is not in the document]
Comment (madpah) [reply public/_locales/en/messages.json:346]: Done for German, Portuguese, Finnish - awaiting French and Korean

File: public/_locales/en/messages.json [hunk 6]
```diff
@@ -331,12 +351,20 @@
...
+        "description": "Used when a product, feature, or function is supported in the current configuration"
+    },
     "GITHUB_STARS": {
         "message": "Stars",
         "description": "The number of GitHub stars for the project"
     },
     "GITHUB_FORKS": {
         "message": "Forks",
         "description": "The number of GitHub forks for the project"
+    },
```
[the remark this answers is not in the document]
Comment (madpah) [reply public/_locales/en/messages.json:357]: Done for German, Portuguese, Finnish - awaiting French and Korean

**Pull request merged**